cartridges/int_bolt_sfra/cartridge/client/default/js/boltProductPageButton.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+// initProductPageCheckoutButton Polls:
+// 1) the DOM for the product page checkout button container
+// 2) BoltConnect
+// when they become available, it attaches an onclick event handler to it.
+var initProductPageCheckoutButton = function () {
+    var boltButtonExist = setInterval(function () {
+        var checkoutBoltButton = $('.bolt-product-checkout-button')[0]; // @ts-ignore
+        // have to check if child of this dom is svg, otherwise bolt button is not fully rendered (it's the object)
+        var boltButtonLoaded = checkoutBoltButton && window.BoltCheckout;
```

Comment:
Do we need to check if the child is `svg` here?

Reply:
You are correct! But I am going to add it in the next PR